gammapy/scripts/cta_sensitivity.py

```diff
@@ -9,6 +9,7 @@
 from gammapy.spectrum.models import PowerLaw
 from gammapy.spectrum.utils import CountsPredictor
 from gammapy.scripts import CTAPerf
+from scipy.optimize import newton
```

Comment:
In Gammapy we still have `scipy` as an optional dependency, and an `import scipy` at the top of this file will fail our CI build where we collect / run tests with just the core dependencies and no scipy installed.
Making `scipy` and `yaml` core dependencies is something we should discuss, but for now, @mackaiver - Please move the import into the method where you use it.

Reply:
done